site/ko/swift/python_interoperability.ipynb

```diff
@@ -0,0 +1,675 @@
QET4u6DW08awmbRwmXWYVXigzfx6GqUyp0JRAioVwxGvRZE3ijcWhSXv\nJ7roZkdq5/udAIC3Z1r7CfpaKo+7qylqO5Wpp0c80GtU+O7EitRHaUpUKJKW47MbEYxR66lInJXr\notbThhnwmBXferqZeOqxSHwSZTrqFZJPlbSeAABuLApBNgNuCrKR2pjXCotBg7cr83ut6q2Zynwi\nBdnIklmvwdMjHnx3YkVxXw9rgQpF0nJ8diNCyRylTlYEIhnoNCp42vRSH0UxBtxtmFlTduupWCgO\nttMTdimMea1Ib5QUF6p0YymOcb+NgmxkQK1iON/nbPkbxTen19Gm1+A4rWORrY8d78RKIof3Kh0H\npHpUKJKW02U3osyBFQq0ASC0nnY5jPTEqYH6PWaspTYQzyp3d9NUOAW/3QiTTiP1URTpaGXZvJLm\nFDeKZdwKJnCym9pO5eLRASdm1tIIJVv3+/GbM+s41+egPcUy9syRDmjVjNpPD4A+q0nL8TuEWbzF\nmLJeSX+YxRjtUGw0se1NyXOKU6EUBmk+UTIjHRaomLKST++uJrFRKtPNjoyc7xfaMS/Ptuat4moi\nh5lwmuYTZc5m1OLCoBsvTayAc2o/3Q8qFEnL6aoUiktRCrQBhBtFCrJprAGPspNPy2WO6XAKQx4q\nFKVi0Kox6GnDTQUVijcW4wCAkxRkIxvjPivMOnXLtp/en090S3wSspfnxzuxEMko6mtiLVChSFqO\n12YEY6AVGQASuQLi2QLdKDZYj9MEtYop9kZxKZZFrlCmxFOJjXmtimo9vbEYg92kpRfGZESjVuGR\nPmfLBtpcmlqH1aDZbPUm8vVjRzugYsD3qP10X6hQJC1Hp1Gh3aKnG0UIQTYA7VBsNJ1GhW6HUbHJ\np1OVm1QqFKU17rciGM8hkt6Q+igNcWMxjuN+GxijeWw5ebTfiburqZb7POSc4/WpNVwYdENNGQCy\n52rT43y/Ey9RobgvVCiSluS3G+lGEVt2KNKNYsP1u82YVmjr6XQl8XSYCkVJjfuEWb2JpbjEJ6m/\nXKGEO6tJnOii+US5ebSyT7HV5hRn19JYimVxcZjaTpvFx8a9uBdK4e5qUuqjNA0qFElL8jtMWKQb\nRSxGxRtFasVqtAFPG+bW04rc2zQVSsFl1sFh1kl9FEU7Vgl1mQi2fqF4czmBUpnjBM0nys6JLjsM\nWlXLtZ++PrUGAHiSCsWm8bHjnVAx4MXrS1IfpWlQoUhakt9uxHI8q8gn6VsFIhlY9BrYjFqpj6I4\nAx4zcoUylhOtGwv/MPco8VQWbEYtepwmRdwo3ggI+9HoRlF+dBoVzvQ4Wi7Q5rW7a+h2GtHrol2x\nzaLdYsATQ268eD1I6adVokKRtCS/w4hCiSOUzEt9FEkFoln4HUaa2ZFAf2VFhtKSTznnmAqlaD5R\nJsb9VkwstX6gzY2lONxtenRaDVIfhezg8QEXbq0kWmZOsVAq462ZdVwc8kh9FLJPnzzlx2I0i3fm\no1IfpSlQoUha0uaKDIXvUgxEMhRkI5HBymoIpSWfrqU2EM8WaDWGTIz7bViIZBDPFKQ+Sl3dWIzj\nZBcF2cjVE8NucA68Od0a7afvBWJI5YvUdtqEPjreCYNWhb+h9tOqSFIoMsacjLG/Z4zdq/zo2OF9\nTjHG3mSMTTLGbjDGfmrL7/0RY2yWMXa98jjV2P8FRO667EKhqOQ5Rc45FqNZCrKRSLtFD7NOrbjk\n06kQJZ7KiRhoM9nCc4qpfBHT4RSOU9upbJ3w22DRazbn+prdj+6tgTHgwqBL6qOQfWrTa/DcWAe+\nfWMZhVJZ6uPInlQ3ir8B4BXO+TCAVyo/3y4D4DOc82MAngfwB4yxrVPqv845P1V5XK//kUkz8W/e\nKCq3UFxLbSBbKFGQjUQYY+j3KC/5lFZjyMu4AgJtJpfi4Bw4SUE2sqVRq/DogAtvtEih+PrUGk74\nbbCbKLCrGX3ylB/RTAGv3Q1LfRTZk6pQfAHANypvfwPAJ7e/A+f8Luf8XuXtIIAQAGoGJ1Ux6TRw\nmLSKvlHcTDylG0XJDLjbFNd6Oh1KwaxTw2ujWTE5cJp18NuNeL+F5xRvLApFMN0oytsTQy4sRDKb\n+32bVSJXwPVADE8O01PSZvXUiAd2kxZ/cz0o9VFkT6pCsYNzvlx5ewVAx27vzBg7D0AHYHrLL/9O\npSX13zPG9Lv82S8wxq4yxq6Gw/TKgZL4HUYsKbhQDFT+t9OMonQGPGYsxbLIFUpSH6Vh7oWSGGxv\no1kxGTnms2KyhZNPbyzF4bcb4W576FMBIgMXh4R5vma/VXxreh2lMqf9iU1Mp1Hh48e9+PubK0jl\ni1IfR9bqVigyxl5mjE3s8Hhh6/txIZ/2oRm1jDEvgD8G8DnOudhM/CUARwCcA+AE/n/27ju8zfu8\n9//7AUBwYZAEuAnuPTQoajuy5SnbsROPkzhulhMnqZM050pXmo78+ktzTt10pOf0nKZJm2Y0O07r\nleER27GtPUmJey9wASTBTYLAc/4ACUm2JGuQfDDu13XpEiU9wHPTF03ig+/3e9984XKPV1X1m6qq\n1qmqWpeaKu/+RJPspPio3nq6+q7tamMfsfEK7ImoKvS4o2dVsW1khtJ0s9ZliAvUZFvpcs0yvRCZ\nDW3O9E/IWIwwUJxmIs0cG/bnFN9odxEfo6c2923tNUQYebA2hwWvn+frZVXxStYtKKqqeruqqtWX\n+PUMMLISAFeD4OilnkNRFAvwC+DPVFU9csFzD6kBi8C3gR3r9XmI8JWdlMDgxHzUzsoZmJjDlmgk\nMdagdSlRK9j5NEoa2kzMLjE2vUhpupxPDCWr5xSbnJG3/dQ1s0j/+Dxbc+V8YqhTFIW9xXYOd7rD\ndsaxqqq82jrK3mIbRoMMDghntblJlKSZ+NHxfq1LCWlafZU/C3xk5eOP
```

Comment:
Swift notebooks are a bit different than Python notebooks. You need to manually set the kernel to use Swift in Colab (there's not a UI option, yet).

See the original notebook (at the bottom): https://raw.githubusercontent.com/tensorflow/swift/master/docs/site/tutorials/python_interoperability.ipynb

I would recommend starting a translation by copying that notebook instead of starting a new one. These *should* execute in Colab.

Reply:
Meta data has been modified for Swift.

Perhaps the kernel changed while checking on local notebook after work in colab

Thank you